src/main/kotlin/com/mparticle/kits/IterableKit.kt

```diff
@@ -205,6 +219,7 @@ class IterableKit : KitIntegration(), ActivityListener, ApplicationStateListener
     }
 
     companion object {
+        var prefersUserId = false
```

Comment:
Should this be private and then have a public setter the same way customConfig does?

Reply:
This attribute is not written from a config object, so I assumed it was set directly. When having a public setter/getter, there is no need of adding those in Kt. If this was written only using a configuration object, I would have made this private set/public get, but thats not the case.

Reply:
Yeah in the iOS kit it's static so it can be set before init

Reply:
Here you'll be able to using the static variable so I think its fine.